app/assets/javascripts/discourse/templates/components/signup-cta.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="signup-cta alert alert-info">
+  {{#if session.hideSignupCta}}
+    <p>
+      {{i18n "signup_cta.hidden_for_session"}}
+      <a {{action "neverShow"}}>{{i18n "signup_cta.hide_forever"}}</a>
+    </p>
+  {{else}}
+    <p>{{i18n "signup_cta.line_1"}}</p>
```

Comment:
I am not a fan of `line_1` and `line_2` for this. I think it would be better if you could come up with a better description for the paragraphs, or perhaps just include both together + HTML for paragraphs in one key.